themes/base.theme.bash

```diff
@@ -207,8 +207,10 @@ function git_prompt_minimal_info {
 }
 
 function git_prompt_vars {
-	if ${SCM_GIT_USE_GITSTATUS} && _command_exists gitstatus_query && gitstatus_query && [[ "${VCS_STATUS_RESULT}" == "ok-sync" ]]; then # use faster gitstatus
-		SCM_GIT_GITSTATUS_RAN=true # use this in githelpers and below to choose gitstatus output
+	if ${SCM_GIT_USE_GITSTATUS} && _command_exists gitstatus_query && gitstatus_query && [[ "${VCS_STATUS_RESULT}" == "ok-sync" ]]; then
+		# we can use faster gitstatus
+		# use this variable in githelpers and below to choose gitstatus output
+		SCM_GIT_GITSTATUS_RAN=true
```

Comment:
Was this change related to the keep-padding setting?

Reply:
Yes, as `shfmt` did a pretty wierd formatting to the comment, so I preferred to change it entirely